tests/test_utils.py

```diff
@@ -107,7 +112,37 @@ def test_detect_average_age():
     result = utils.get_average_risk_score(domains)
     assert result == None
 
+
 def test_data_prune():
     data = espn()
     utils.prune_data(data)
     assert data == prune_espn_expected()
+
+
+def test_find_emails():
+    emails = utils.find_emails(json.dumps(espn()))
```

Comment:
@ChuckWoodraska I don't know about using ESPN as a published test domain. What do you think about using domaintools.com since we don't bill for those queries?

Reply:
We can change it, but its not actually using queries. Its a saved data response. See iris_investigate_data.py in tests/responses.